rocketpy/EnvironmentAnalysis.py

```diff
@@ -610,11 +612,148 @@ def calculate_average_day_wind_rose(self):
         """average day wind rose"""
         ...
 
+    def plot_wind_gust_distribution_over_average_day(self):
+        """Plots shown in the animation of how the wind gust distribution varies throughout the day."""
+        ...
+        # Gather animation data
+        average_wind_gust_at_given_hour = {}
+        for hour in list(self.surfaceDataDict.values())[0].keys():
+            wind_gust_values_for_this_hour = []
+            for dayDict in self.surfaceDataDict.values():
+                try:
+                    wind_gust_values_for_this_hour += [dayDict[hour]["surfaceWindGust"]]
```

Comment:
You can substitute this try/except block by:

`[dayDict.get('hour', {}).get('surfaceWindGust', None)] if dayDict.get('hour', {}).get('surfaceWindGust', None) else []`

But honestly it is not the most readable sentence. So if you want to maintain the try/excepet block ok, but there is the option to change for this sentence, up to you